core/src/modules/ops/import.ts

```diff
@@ -66,11 +68,22 @@ export namespace ImportOps {
   }
 
   export async function associateProfile(_import: Import) {
-    const {
-      profile,
-      isNew,
-      // will throw if there are no unique profile properties
-    } = await ProfileOps.findOrCreateByUniqueProfileProperties(_import.data);
+    let run: Run;
+    let schedule: Schedule;
+    let source: Source;
+
+    run = await Run.findOne({ where: { id: _import.creatorId } });
```

Comment:
We're probably ok since the other types aren't really used, but maybe also check that the import's creatorType refers to a run?